src/main/java/com/novauc/SpringBasicsFRController.java

```diff
@@ -0,0 +1,85 @@
+package com.novauc;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.http.HttpSession;
+import java.util.ArrayList;
+
+/**
+ * Created by octavio on 3/14/17.
+ */
+
+@Controller
+public class SpringBasicsFRController {
+
+    // crete a route for "/"
+    // a) it should take the model and request as arguments // {args}
+    // b) it should read the username from the session and add it to the model add.
+    // c) it should return the home template  return;
+
+//    @RequestMapping(path = "/", method = RequestMethod.GET)
+//    public String home(Model model, HttpSession session) {
+//        model.addAttribute("name", session.getAttribute("userName"));
+//        return "home";
+
+
+    @RequestMapping(path = "/", method = RequestMethod.GET)
+    public String home(Model model, HttpSession session) {
+        model.addAttribute("name", session.getAttribute("userName"));
+        return "home";
+    }
```

Comment:
Displaying messages on the html does not work.  Please update your code so that the html displays properly. REMEMBER if you want something to show up in html you have to give that value to the model.  Why does name show up and message does not?